### PR TITLE
Ftest

### PR DESCRIPTION
**Pull request by Fede-edef**
*Adding tests for mgPipe*

**I hereby confirm that I have:**

- [x ] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [ x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [ ] with [X] to check the box)*

### CODE REVIEW

File: test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m [hunk 1]
```diff
@@ -134,3 +139,78 @@
...
+
+% test createdModels
+[createdModels]=createPersonalizedModel(abunFilePath,resPath,setup,sampName,organisms,indNumb);
+assert(length(createdModels)== 5)
+assert(strcmp(createdModels(2,1),'Test1') == 1)
+assert(strcmp(createdModels(5,1),'Test4') == 1)
+
+microbiota_model=load(strcat(resPath,'microbiota_model_samp_Test1')); 
+microbiota_model=microbiota_model.microbiota_model;
+assert(length(microbiota_model.A)> length(microbiota_model.S))
```
Comment (tpfau) [inline test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m]: I would recommend not testing this (A/csense/S), as it is likely to become invalid in the future. 
In particular, as to the model definitions, length(csense) should be size(S,1). 

Also I would suggest to use an explicit size(S/A,1/2) to refer to the respective dimensions, as length can be either dimension and if someone else tries to figure out what a certain assert in a test means, this makes it really hard to detect potential problems.

File: test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m [hunk 2]
```diff
@@ -134,3 +139,78 @@
...
+assert(length(microbiota_model.csense) == length(microbiota_model.A))
+assert((length(strmatch('communityBiomass',microbiota_model.rxns)) > 0) == 1)
+assert((length(strmatch('UFEt_microbeBiomass',microbiota_model.rxns)) > 0) == 1)
+assert((length(strmatch('EX_microbeBiomass[fe]',microbiota_model.rxns)) > 0) == 1)
```
Comment (tpfau) [inline test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m]: May I suggest:
`any(ismember('EX_microbeBiomass[fe]',microbiota_model.rxns))` ?
It is equivalent, but from local tests, `any`+`ismember` is faster and (at least personally) I find it easier to understand what is tested.

File: test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m [hunk 3]
```diff
@@ -134,3 +139,78 @@
...
+assert(length(microbiota_model.rxns(strmatch('DUt',microbiota_model.rxns)))== length(microbiota_model.rxns(strmatch('UFEt',microbiota_model.rxns)))-1 == 1)
```
Comment (tpfau) [inline test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m]: This assertion is odd (imo). 
It tests that the following is true:
`'DUt'` must be in `microbiota_model.rxns` and `'UFEt'` must be in `microbiota_model.rxns`
But the way this is tested is really odd...

File: test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m [hunk 4]
```diff
@@ -134,3 +139,78 @@
...
+assert(exist('fixVec', 'var') == 1)
+assert(length(organisms) == 5)
+
+% test fastSetupCreator
+
+setup=fastSetupCreator(models, organisms, {},objre);
+ 
+assert(exist('setup', 'var') == 1)
+assert(strcmp(class(setup),'struct') == 1)
+assert((length(setup.S) == length(setup.rxns)) == 1)
```
Comment (tpfau) [inline test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m]: As further down:
could you please use `size()` for matrices?
Also:
Could `verifyCobraModel` make sense here?

**Comment by laurentheirendt** [comment]
Thanks @Fede-edef 👍

**Pull request merged**